code/Location.php

```diff
@@ -130,9 +125,22 @@ public function validate()
         return $result;
     }
 
+    public function EmailAddress()
+    {
+        Deprecation::notice('2.0', 'Use "$Email" instead.');
```

Comment:
Can you update the deprecation to `'3.0`. After some discussions I recently had, this technically wouldn't go away until we release `3.0` and `2.x` will continue to support this. We'll need to do this for any other deprecations going into `master`.